scripts/render.js

```diff
@@ -9,15 +9,15 @@ Run as `npm run render $query $depth $aggregateMode`
 
 const bcd = require('..');
 
-var query = process.argv[2];
```

Comment:
Since we're not actually using this script anymore, may we revert all changes to this file?